src/test/java/uk/gov/justice/hmpps/prison/dsl/DateHelper.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.hmpps.prison.dsl
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+@Suppress("ktlint:standard:function-naming")
+// add more months when required
```

Comment:
silliness to avoid LocaDateTime.parse() to annoy @mikehalmamoj

Reply:
It's beautiful 😡